WooCommerce/Classes/ViewRelated/Orders/Order Details/Shipping Labels/WooShipping Create Shipping Labels/WooShipping Split Shipments/WooShippingSplitShipmentsRow.swift

```diff
@@ -0,0 +1,49 @@
+import SwiftUI
+
+struct WooShippingSplitShipmentsRow: View {
+    @State private var isShowingDetailView = false
+
+    let viewModel: WooShippingSplitShipmentsViewModel
+
+    init(viewModel: WooShippingSplitShipmentsViewModel) {
+        self.viewModel = viewModel
+    }
+
+    var body: some View {
+        AdaptiveStack {
+            Text(Localization.products)
+                .tertiaryTitleStyle()
+            Spacer()
+            Button(Localization.splitShipments) {
+                isShowingDetailView = true
+            }
+            .buttonStyle(TextButtonStyle())
+        }
+        .padding(.vertical, Layout.verticalPadding)
+        .sheet(isPresented: $isShowingDetailView) {
```

Comment:
Nit: `sheet` should be replaced with `fullScreenCover` to better match the design.

Reply:
TIL about `fullScreenCover`. Thanks! Done in https://github.com/woocommerce/woocommerce-ios/pull/15369/commits/da2e8b2a1b59d8b7bb2c85ba6f6ca4bca3472ae6